src/agentscope/agents/text_to_image_agent.py

```diff
@@ -46,15 +45,22 @@ def __init__(
     def reply(self, x: dict = None) -> dict:
         if self.memory:
             self.memory.add(x)
-
+        if x is None:
+            # get the last message from memory
+            if self.memory.size() > 0:
```

Comment:
This may trigger error when `use_memory=False`. Maybe can be
```
if self.memory:
  # because latest message or x must be added to memory already at line 46
  x = memory.get_memory()[-1]
elif x is None:
  # no memory and x is None, returns an empty message?
  return {}
```